cogs/fun.py

```diff
@@ -1,6 +1,10 @@
+import aiohttp, urllib.parse
```

Comment:
Are either of these libraries we'll need to add to requirements.txt?

Reply:
aiohttp was already in the requirements, and urllib is a default python library.